cf-kube-dist/cf-kube-dist.yml

```diff
@@ -6,15 +6,15 @@ resources:
   source:
     branch: develop
     private_key: {{github-private-key}}
```

Comment:
That means we no longer need the private key, right? (Since we're not using ssh anymore)